src/meshapi/util/uisp_import/update_objects.py

```diff
@@ -104,7 +106,7 @@ def update_link_from_uisp_data(
 
     if uisp_link_id != existing_link.uisp_id:
         existing_link.uisp_id = uisp_link_id
-        logging.info(
+        change_messages.append(
```

Comment:
You don’t want this. It’s super spammy. Like 10 messages an hour spammy

Reply:
I'll set dirty bit on this, then.